pkg/ingestor/parser/internal/mock/types.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2023 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Code generated by MockGen. DO NOT EDIT.
+// Source: pkg/ingestor/parser/common/types.go
+
+// Package mocks is a generated GoMock package.
+package mocks
```

Comment:
Could you move this into `internal/testing` or a subfolder in `internal/testing/mocks` to keep it all in once place?

Reply:
Moved the file to `internal/testing/mocks`.